packages/react/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  setLocalStorageItem,
+  removeLocalStorageItem,
+  LocalStorageEventPayload,
+  createLocalStorageChangeEvent,
+} from '@lubycon/utils';
+
+import { useEffect, useState, useCallback } from 'react';
+
+function tryParse(value: string) {
+  try {
+    return JSON.parse(value);
+  } catch {
+    return value;
+  }
+}
+
+function isTypeOflocalStorageChangeEvent<T>(
+  evt: any
+): evt is CustomEvent<LocalStorageEventPayload<T>> {
+  return !!evt && evt.type === createLocalStorageChangeEvent.eventName;
```

Comment:
마이너) 요런 암시적 형변환보다 명시적 형변환이나 명확한 조건문을 사용하는 것이 더 가독성이 좋다고 생각하는데 어떠신가유 👀 

```suggestion
  return Boolean(evt) && evt.type === createLocalStorageChangeEvent.eventName;
```
```suggestion
  return evt != null && evt.type === createLocalStorageChangeEvent.eventName;
```